docs/concepts/explore-available-balancer-pools/reclamm-pool/reclamm-pool.md

```diff
@@ -0,0 +1,124 @@
+
+## Price range mechanism
+
+One fundamental thing to understand is how the price range is defined and enforced in the first place, given that the price curve is essentially "weighted math" (constant product), and the price is normally determined by the token balances, which are unconstrained (beyond needing to be greater than 0).
+
+The answer is the introduction of "offsets" to the real balances called "virtual" balances, such that the token balances used to calculate the invariant are redefined as the sum of the real and virtual balances. These virtual balances fix the price curve on both ends, cutting off the long tail and ensuring non-zero minimum and maximum prices, even as the real balances approach zero. (This mechanism is shared by Gyro pools, but the terminology here is very slightly different.)
+
+Higher virtual balances (relative to the real balances) means higher concentration. 
+
+## Initialization
+
+What we really need internally is the "ratio" of the minimum and maximum prices, but since calculating this is unintuitive and a lot to ask of integrators, we created an initialization mechanism that takes very simple input: the actual minimum and maximum prices, and a target price. Two additional parameters are required to specify the behavior of the pool after initialization (margin and price shift exponent); those are described below.
+
+One question that arises immediately is: how do we define the price? There are after all *two* ways to define the price of a 2-token pool: A in terms of B, or B in terms of A. We have chosen to define the prices as B/A; i.e., prices represent the value of token A denominated in token B. In other words, how many B tokens equal the value of one A token. Since Balancer pool tokens must be registered in numerical order, the "direction" of the ratio is deterministic.
+
+When the pool is created from the factory, these initial values are stored immutably; they are only used during initialization.
+
+The initialization process essentially derives the "internal" parameters from the initial user input, and given the real token balances being supplied, checks that the resulting prices match their intended values: otherwise the pool would be vulnerable to losses through arbitrage.
+
+See the link below for the detailed math, but we start with the definition of the invariant:
+
+$L = (R_a + V_a)(R_b + V_b)$
+
+And the price, recalling that it is defined as B/A:
+
+$P_a = \frac{R_b + V_b}{R_a + V_a}$
+
+We know the desired price *ratio* (max/min), and we see that these values can be derived by setting each real balance to 0. At the edges, one of the balances will be the maximum value, and the other will be 0. The "maximum balance" is really a placeholder needed for intermediate calculations (it will cancel out later), so we can assign it a convenient arbitrary value (e.g., 1000). Given the invariant and these price bounds, we can derive initial values for the virtual balances.
```

Comment:
As mentioned above, the maximum balance is not just a placeholder, in the sense that it has a real meaning. But, in this paragraph it's clearer that the arbitrary value is the placeholder, and not the concept of a maximum value.